pallets/time-release/src/tests.rs

```diff
@@ -557,18 +615,23 @@ fn alice_time_releases_schedule() {
 				.into(),
 		);
 		assert_ok!(TimeRelease::claim(RuntimeOrigin::signed(BOB)));
-		assert_eq!(PalletBalances::locks(&BOB).first(), None);
+		assert_eq!(
+			<Test as Config>::Currency::balance_frozen(&FreezeReason::NotYetVested.into(), &BOB),
+			0
+		);
 	});
 }
 
 fn get_balance<T: Config>(who: &T::AccountId) -> BalanceOf<T> {
-	T::Currency::free_balance(who)
+	T::Currency::balance(who)
 }
 
 fn set_balance<T: Config>(who: &T::AccountId, balance: BalanceOf<T>) {
-	let deposit_result = T::Currency::deposit_creating(who, balance.saturated_into());
-	let actual_deposit = deposit_result.peek();
-	assert_eq!(balance, actual_deposit);
+	let _ = T::Currency::mint_into(&who, balance.saturated_into());
+	assert_eq!(
+		T::Currency::balance(who).saturated_into::<u64>(),
+		balance.saturated_into::<u64>() + 100u64
```

Comment:
Why is this 100 here?

Reply:
Existential balance, I'd guess?

Reply:
ED is what I was thinking when I put it there, _but_ this runs in the tests and so ED is more like `1`.
The real reason is that it was refactored incorrectly.  The new code works a bit differently.